db/seeds.rb

```diff
@@ -5,3 +5,73 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'csv'
+
+POLLS = 1000
+USERS = 10000
+TIMEFRAMES = 20000
+COMMENTS = 10000
+
+poll_data = CSV.read(Rails.root.join('db', 'seeds', 'poll_seed.csv'), {col_sep: ';'})
```

Comment:
The file poll_seed.csv, is this for the tsung load testing? 
In file workflow_woCSRF.xml, the file opened seems to have a different name -- poll_details.csv

Reply:
No, those are two different files.